DQM/SiStripMonitorClient/test/test_TreeAndTkMapProducer.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+function die { echo $1: status $2; exit $2; }
+RUN="319176"
```

Comment:
technically you don't need to know the run number to run these tests.

Reply:
Yes, you are right @mmusich , it's only used in the printout. I would rather not change this now, as I'd like the backport of this to be part of the 12_4_X that we cut today. Is that ok with you? (If we dont cut a release today, I can change it)

Reply:
From ORP: we have seme time for 12_4_X, so I changed this now